helpers/publicarMovimientoDetalle.go

```diff
@@ -104,14 +105,15 @@ func PublicarMovimientosDetalle(idMovProcExterno int) (movimientosDetalleRespues
 		movimientosDetalleRespuesta = registroCuentas
 	}
 
-	// logs.Debug(fmt.Sprintf("registroCuentas: %+v", registroCuentas))
+	logs.Debug(fmt.Sprintf("registroCuentas: %+v", registroCuentas))
```

Comment:
Dejar comentado o eliminar
```suggestion
	// logs.Debug(fmt.Sprintf("registroCuentas: %+v", registroCuentas))
```